content/spin/v2/install.md

```diff
@@ -286,10 +286,74 @@ Then install plugins by name.
 
 Python:
 
+> Historic: You may have see a `py2wasm` plugin in your travels. Please note, `py2wasm` (an experiment to build a Spin Python SDK using CPython, Wizer, and PyO3) has since been replaced by `componentize-py`. 
+
+The process of getting your system ready to write Wasm-powered Python applications, using `componentize-py` is as follows:
+
+<!-- @selectiveCpy -->
+
+```bash
+$ spin templates install --git https://github.com/fermyon/spin-python-sdk --update
+```
+
+Once we have the above `spin-python-sdk` installed we can scaffold out a new app using the `http-py` template. The scaffolded app has a `requirements.txt` file that facilitates the installation of `spin-sdk` and `componentize-py`. While you could manually install these using Pip, the `requirements.txt` file has the appropriate version numbers set making the process quicker and also more robust. Let's create a new Spin app and run harness the `requirements.txt`:
+
+<!-- @selectiveCpy -->
+
+```bash
+# We then create our app using http-py
+$ spin new -t http-py hello-python --accept-defaults
```

Comment:
We should not be asking people to create an application as part of Spin installation.

Reply:
Yeah, not ideal I guess. The `requirements.txt` is only accessible once the app is created. Alternatively, we could just tell them to install what is in the `requirements.txt` file (point to GitHub URL).

Reply:
I'm not sure even that would be helpful.  The build workflow seems to be predicated around having it installed _in the venv_.  I'd rather just tell people to install things at that point.  Just because py2wasm was a global install that could be done at Spin install time doesn't mean we have to treat `componentize-py` as a global install-time install.  It's a different workflow from the plugin-based one: we don't need to try to fit it into the same shape if that makes sense?

Reply:
That said it may be best to wait for Joel or Karthik to weigh in.  It may turn out that I have catastrophically misunderstood the workflow and timing!

Reply:
I would be in favour of not installing `componentize-py` when the user just wants to install `spin`. It should happen as a part of the process of creating and setting up a new application. 
The venv approach above also means it could potentially guide the user towards the idea of reusing the same venv for multiple projects which is not correct.  

Each new project must have its own venv and associated dependencies as that way different apps can have their own versions of the dependencies and they stay isolated.

Reply:
Thanks @itowlson and @karthik2804 - that is super helpful.
Please see [latest commit](https://github.com/fermyon/developer/pull/1264/commits/5f58d2fd643827b2764d49fdcf2f62fa55ec668f).